### PR TITLE
Add setting to disable input device grabbing

### PR DESCRIPTION
**Pull request by spiiroin**

### CODE REVIEW

File: tools/mcetool.c [hunk 1]
```diff
@@ -5719,6 +5747,21 @@ static const mce_opt_t options[] =
                         "set the delay for dumping core from unresponsive lipstick\n"
         },
         {
+                .name        = "set-input-policy-mode",
+                .with_arg    = xmce_set_input_policy_mode,
+                .values      = "enabled|disabled",
+                .usage       =
+                        "allow/deny grabbing of input devices based on input policy\n"
+                        "\n"
+                        "Normally this should be always set to 'enabled'.\n"
```
Comment (sledges) [inline tools/mcetool.c]: what undesired effect may users experience if this flag is set to "enable" in a production-ready release?

Comment (spiiroin) [inline tools/mcetool.c]: When display is powered up, the touch input grab is kept until mce thinks
a) the display is no longer touched by finger or palm (jolla has palm detection)
b) proximity sensor is not covered (optional)
c) lid sensor says the display is not covered (nexus5 etc)

In practice would:
* make stray input during in-call blanking more possible when proximity sensor is briefly uncovered while the device is held at ear.
* break the "lpm" mode (where only mce is supposed to handle touch events)

Comment (spiiroin) [inline tools/mcetool.c]: Also depending on what happens on resume, the touch input can get activated well before anything shows up on screen -> we do not want any taps / swipes to go through before state sync between mce & compositor is finished and there is something on screen for user to tap/swipe.

Comment (sledges) [inline tools/mcetool.c]: thanks, asked this in order to avoid false alarms during such tests

**Comment by sledges** [comment]
LGTM

**Pull request merged**